a3p-integration/proposals/a:upgrade-next/package.json

```diff
@@ -6,6 +6,8 @@
     "upgradeInfo": {
       "coreProposals": []
     },
+    "submission": "probe-zcf-bundle",
+    "probe-zcf-bundle": "a:upgrade-next probeZcfBundle probe-submission",
```

Comment:
`a:upgrade-next` is implied by the proposal package.

Reply:
Let's also be clear that this isn't part of the synthetic-chain API.
```json
  "sdk-generate": [
    "probe-zcf-bundle probeZcfBundle probe-submission"
  ]
```

Reply:
done